clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1513,19 +1509,6 @@ private void initControllerClient(String storeName, Optional<SSLFactory> sslFact
     } else {
       LOGGER.info("System store controller client has already been initialized");
     }
-
-    if (pushJobSetting.livenessHeartbeatEnabled) {
```

Comment:
It seems we are not respecting this config anymore. I think it is beneficial to have a flag to start with this feature turned off (the HB store is not yet in AA mode)

Reply:
Yeah, I should not delete this flag. I will add back.